### DISCUSSION TITLE
cannot see users in MMC

### DISCUSSION THREAD

**Discussion by germanlagomarsino** [neos-sdi/adfsmfa#278] [General]
Hello, first I want to congratulate you for developing this tool, it is very good!

The query is because when I create users either by powershell or by the mmc I can't see them in the mmc I can only see them using powershell.

Any idea what could be going on and how to fix it?

thank you!

Regards!

![image](https://user-images.githubusercontent.com/52166710/215532812-d518a71b-1bec-4d42-be7f-6bcc51279254.png)

**Comment by redhook62** [comment]
Hi @germanlagomarsino 

No, no specific idea.

Check that your users are active with PowerShell, and clear the filter in the console (Filter Users, Clear Users Filter).

Otherwise you have a problem on your System.

regrads

**Reply by germanlagomarsino** [reply]
@redhook62 cleaning the filter worked, thank you very much!
